src/Linear/Linear.tsx

```diff
@@ -155,20 +165,38 @@ export default class Linear extends React.Component<LinearProps> {
       }
 
       blockHeights[i] = blockHeight;
+      blockWidths[i] = blockWidth;
+      if (blockHeight > maxBlockHeight) {
+        maxBlockHeight = blockHeight;
+      }
+      if (maxTranslationRowSize < translationRows[i].length) {
+        maxTranslationRowSize = translationRows[i].length;
+      }
+      if (maxAnnotationRowSize < annotationRows[i].length) {
+        maxAnnotationRowSize = annotationRows[i].length;
+      }
+    }
```

Comment:
I've only skimmed this so far -- so I'm not saying this with certainty -- but do we need to calculate the max at all @bugzpodder. I would think that down in `SeqBlock` we could just use the size of the first (and only) row every time

Reply:
The reason for using max is actually for exactly this reason:
https://github.com/Lattice-Automation/seqviz/pull/205#issuecomment-1537213475

The bps and the yellow bar might not match up because of differing translation/annotation.  See this screenshot.

<img width="1607" alt="Screenshot 2023-05-06 at 8 56 52 PM" src="https://user-images.githubusercontent.com/14841421/236656966-0951dedc-f37f-4196-8f02-e092bf00f329.png">

This sequence is what's being used in the current [demo](https://tools.latticeautomation.com/seqviz/)
At bps 1170 you'll see the amino acid translation taking two rows.  If you feel this is ok then then I can take out the maxTranlsationRowSize/maxAnnotationRowSize calculations.

Also, in case you are serious the reason why they don't match up in this particular sequence is because of the translations in the demo:
```
    translations: [
      { direction: -1, end: 630, start: 6 },
      { end: 1147, start: 736 },
      { end: 1885, start: 1165 },
    ],
    ```
    Note that here the second translation section and third section doesn't actually overlap (1147/1165) but they happen to be in the same SeqBlock but the code did not figure out that they happen to be on the same row.